### PR TITLE
Update README to add Transcriber permissions info and remove beta language

### PR DESCRIPTION
**Pull request by becki-at-luminal**
This PR updates the README:

- Update list of commands to match what's on the docs site
- Remove "beta" language
- Emphasize that Fugue installs the policies automatically
- Explain that Transcriber does not use these policies and link to docs site for more info

### CODE REVIEW

**Comment by fugue-carrie** [comment]
Editorial LGTM 🌮

File: README.md [hunk 1]
```diff
@@ -1,13 +1,16 @@
-# (Beta!) fugue-user-iam-templates
-User and installer IAM templates for Fugue.
+# fugue-user-iam-templates
```
[the remark this answers is not in the document]
Comment (alex-luminal) [reply README.md]: I agree. I'll cut a PR against the PR

**Comment by alex-luminal** [comment]
PR in #15 adds deprecation notice to this PR

**Comment by wayne-luminal** [comment]
👍  after Alex's PR is merged in

**Pull request merged**